docs/core/diagnostics/debug-threadpool-starvation.md

```diff
@@ -0,0 +1,312 @@
+---
+title: Debug Threadpool Starvation
```

Comment:
Should this be `ThreadPool`, instead of `Threadpool`?

Reply:
Probably yes. Looking around I've seen ThreadPool, Threadpool, Thread Pool, and Thread pool, but "Threadpool" seems to be the least common variant. My best guess is that ThreadPool is the best choice for the proper noun and "thread pool" is the generic term.